src/index.ts

```diff
@@ -23,16 +23,16 @@
 // Providers:
 import * as analytics from './providers/analytics';
 import * as auth from './providers/auth';
-
-import * as apps from './apps';
-import { handler } from './handler-builder';
 import * as crashlytics from './providers/crashlytics';
 import * as database from './providers/database';
 import * as firestore from './providers/firestore';
 import * as https from './providers/https';
 import * as pubsub from './providers/pubsub';
 import * as remoteConfig from './providers/remoteConfig';
 import * as storage from './providers/storage';
+
+import * as apps from './apps';
```

Comment:
Think this formatting change slipped through?

Reply:
I moved it on purpose. I added one export to this file. I noticed the comment above the block where I moved this line from says “providers”, and this is not a provider. I can extract it to a separate PR if you feel like it.

Reply:
Got it. Would tslint complain about this because it's now out of order?

Reply:
No, TSLint works on blocks of imports - and we have a newline above.

Reply:
Ack.